packages/app-store/zapier/components/TemplateCard.tsx

```diff
@@ -4,10 +4,10 @@ import { Template } from "./AppSettings";
 
 export default function TemplateCard({ template }: { template: Template }) {
   return (
-    <div className="rounded-md border border-gray-200 p-4 ">
-      <div className="flex items-center justify-center">
+    <div className="rounded-md border border-gray-200 bg-white p-4">
```

Comment:
should be `p-5`

Reply:
@CarinaWolli you can also use the recommend feature of GitHub